client/stylesheets/_modal.scss

```diff
@@ -902,6 +902,19 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	height: 100px;
 }
 
+.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"], .modFeedbackInitial button[data-target="#feedbackBuyerNeverModal"] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector `modFeedbackInitial` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line